src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -1366,6 +1368,14 @@ static int32_t cc_replay_startrecord(InteractiveConsole& console, const argument
 
     std::string name = argv[0];
 
+    if (name.size() < 5 || name.substr(name.size() - 5) != ".sv6r")
```

Comment:
Extension check is not case insensitive.

Reply:
done